rlclientlib/vw_model/pdf_model.cc

```diff
@@ -0,0 +1,45 @@
+#include "pdf_model.h"
+#include "err_constants.h"
+#include "object_factory.h"
+#include "sampling.h"
+#include "ranking_response.h"
+#include "trace_logger.h"
+#include "str_util.h"
+
+namespace reinforcement_learning { namespace model_management {
+
+  pdf_model::pdf_model(i_trace* trace_logger) :
+    _trace_logger(trace_logger), _vw(new safe_vw("--json --quiet --cb_adf")) {
```

Comment:
why is json passed?

Reply:
To be able to use the json parser to parse the joined event and grab the PDF out of it.